include/igl/copyleft/cgal/wire_mesh.cpp

```diff
@@ -62,6 +62,20 @@ IGL_INLINE void igl::copyleft::cgal::wire_mesh(
     v = v-c*(PV.rows());
     p = v;
   };
+
+  // Count each vertex's indicident edges.
+  std::vector<int> nedges(WV.rows());
+  for(int v = 0;v<WV.rows();v++)
```

Comment:
This loop is not necessary. The constructor for `std::vector` with a size parameter will initialize all the values to their default (0 for int). Take a look at constructor (2) in https://en.cppreference.com/w/cpp/container/vector/vector.

Reply:
I didn't realize std::vector did this. Thanks!

Reply:
Personally I prefer to put `0` for the second argument to be more explicit (it doesn't hurt), but that's just my preference.

Reply:
I've added the 0.